input/choice/src/commonMain/kotlin/majestic/DumbSelect.kt

```diff
@@ -37,6 +39,8 @@ fun <T> DumbSelect(
     modifier: Modifier = Modifier,
     containerColor: Color = Color.Unspecified,
     shape: Shape = MenuDefaults.shape,
+    drawerContainerColor: Color = Color.Unspecified,
+    drawerShape: Shape = MenuDefaults.shape,
```

Comment:
Why is there a drawer in a select tag? I think I kind of understand what you are going for, but calling it drawerXXX seems to not be the right name